Tests/Parser/fixtures/oss.yml

```diff
@@ -4142,3 +4142,43 @@
     version: "14.4"
     platform:
     family: Mac
+-
+  user_agent: Aloha/1 Mana/0.0.1 Darwin/21.0.0
+  os:
+    name: iOS
+    short_name: IOS
+    version: "15.0"
+    platform:
+    family: iOS
+-
+  user_agent: BrightSign/8.3.23 (XT1144) Mozilla/5.0 (X11; Linux aarch64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.3 Chrome/69.0.3497.128 Safari/537.36
+  os:
+    name: BrightSign
```

Comment:
Looks like official name is `BrightSignOS` https://www.brightsign.biz/ecosystem/brightsign-os/